docs/components/list-group.md

```diff
@@ -31,16 +31,16 @@ Add labels to any list group item to show unread counts, activity, etc.
 {% example html %}
 <ul class="list-group">
   <li class="list-group-item">
-    <span class="label label-default label-pill pull-xs-right">14</span>
     Cras justo odio
+    <span class="label label-default label-pill pull-right">14</span>
```

Comment:
1.  I'm confused about pull-right/left, because it is still there in on v4-dev tip and I'm not seeing the responsive classes.  I can change that back.
2.  It does not appear to have any problems floating in my test (granted it may be limited testing considering I didn't go after a bunch of browsers).  The fact is that `pull-right` appears to work this way in the docs for both flex and normal displays.

I can change it back, but then we may have questions about a flexed pull-right not working too...

I'll change the pull-*-right back and commit.